client/components/GameList.jsx

```diff
@@ -3,13 +3,13 @@ import GameListEntry from './GameListEntry.jsx';
 
 const GameList = props => (
   <div className="game-list" >
-    <div className="game-list-heading">List of Related Games!</div>
     {props.games.map((game, index) => (
       <GameListEntry 
         game={game} 
         key={index}
         handleGameMouseEntry={props.handleGameMouseEntry}
         handleGameMouseExit={props.handleGameMouseExit}
+        selectedGame={props.selectedGame}
       />
     ))}
```

Comment:
Might want to do the same thing here as I recommended in the `GameDetailsSideBar.jsx` file where you assign the map results to a variable, but I think in this case this is concise enough and doesn't include a conditional so you're probably fine if you want to keep it this way.

Reply:
deferred change